### ISSUE TITLE
i686: Missing ffmpeg 5.1.2-7

### ISSUE THREAD

**Issue by lvalibaba** [talss89/termux-home-assistant-installer#2]
I have tablet Samsung. Installing process marked error with ffmpeg packages. 

dpkg: error processing archive ./contrib/ffmpeg_5.1.2-7_aarch64.deb (--install):
 package architecture (aarch64) does not match system (i686)
Errors were encountered while processing:
 ./contrib/ffmpeg_5.1.2-7_aarch64.deb

**Comment by talss89** [comment]
Thanks for raising the issue. 

What model is your tablet, and how old is it?

It's surprising to see `i686` being requested - that's from the 32-bit Pentium era.

That said, it shouldn't be too difficult to build packages for it.

**Comment by lvalibaba** [comment]
Very old model: Samsung Galaxy Tab 3 10.1 P5220, with custom firmware updated to Android 7.1.2 (max possible).

**Comment by talss89** [comment]
Thanks. You could probably install a normal Linux distro onto this device, and run full Home Assistant with Docker.

I'll look into providing i686 packages, and will update this issue.

**Comment by lvalibaba** [comment]
After reviewing the installation script, I think that there will be problems with the numpy package. I'm new to all this, so I can't fix the setup myself.

**Comment by lvalibaba** [comment]
> Thanks. You could probably install a normal Linux distro onto this device, and run full Home Assistant with Docker.
> 
> I'll look into providing i686 packages, and will update this issue.

If only I knew how to install a full installation. :D

**Comment by talss89** [comment]
You're right, `numpy` is also only built for `aarch64` - that would be the next problem after `ffmpeg`.

I'm working at my 'real job' right now, but will see if I can build you some i686 packages for `numpy` and `ffmpeg` later. Interested to see if we can get this working.

**Comment by lvalibaba** [comment]
Previously, I installed numpy simply: `MATHLIB=m pip install numpy==1.23.2`.
It would be interesting to try a full installation, but I did not find installation packages for my device.
I also did not understand how you did with `lru-dict`, which is written in an interesting way. I installed version `2023.3.6` by recompiling the wheel in which I changed the `lru` version from `1.1.8` to `1.1.6`.

With new versions of Core, everything worked, except for `ha-av` requiring `ffmpeg`, which I could not downgrade.
